src/geom/SpanSet.cc

```diff
@@ -99,6 +99,72 @@ bool spansContiguous(Span const& a, Span const& b, bool compareY = true) {
                    : false;
 }
 
+/* Determine the intersection with a mask or its logical inverse
+ *
+ * spanSet - SpanSet object with which to intersect the mask
+ * mask - Mask object to be intersectedNot
+ * bitmask - bitpattern to used when deciding pixel membership in the mask
+ *
+ * Templates:
+ * T - pixel type of the mask
+ * invert - boolean indicating if the intersection is with the mask or the inverse of the mask
+            templated parameter to ensure the check is compiled away if it can be
```

Comment:
I don't feel too strongly about that as this function can only be called w/n this file, and I like not having a default in that case, as it provides symmetry in readability at the point it is called in code.